lms/static/styles/components/_browser-check-warning.scss

```diff
@@ -0,0 +1,22 @@
+@use '../variables' as var;
+
+// A warning banner shown at the top of the view in unsupported browsers.
+// This is styled to match warning toasts and banners in the Hypothesis client.
+.browser-check-warning {
```

Comment:
Good job on balancing having to hard-code CSS here (because there aren't the same shared mixins as in the client) versus adding a few targeted variables.